layers/+tools/kubernetes/packages.el

```diff
@@ -27,7 +27,7 @@
   '(
     kubernetes
     kubernetes-evil
-    kubernetes-tramp))
+    (kubernetes-tramp :toggle (version< emacs-version "29.1"))))
```

Comment:
```suggestion
    (kubernetes-tramp :toggle (version< emacs-version "29.0"))))
```

there is no Emacs 29.1 yet.

Reply:
I'll fix it. Same fix was done for docker-tramp so just used that one.

Reply:
Some people run master, myself included.

Reply:
Actually this also catches 30+ which is what emacs master is at.

Reply:
see https://github.com/emacs-mirror/emacs/tree/a259d0dda3878a64373b808627c4b4cab3971194

it's been there since 29.0.5

Reply:
This still fixes the problem.

Reply:
This still fixes the problem. 

What I'm saying is that this catches Emacs 30+

Reply:
Okay here's how to reproduce it: Run emacs snapshot and have the kubernetes layer. 

Let's pick a version threshold. Sorry for coming off brash but there was a very similar fix committed to the docker layer.

Reply:
https://github.com/syl20bnr/spacemacs/commit/50825419ab86983e0019c1142d548b0389c50c01

Reply:
Here's the PR you approved: https://github.com/syl20bnr/spacemacs/pull/15899

Reply:
I will fix that.

Reply:
What value should I put? 29.0?

Reply:
I'll do it: what value should I use?

Reply:
Actually: 

See: https://www.emacswiki.org/emacs/TrampAndDocker

> NOTE: if running Emacs built from the sources (or v29.1), as of Nov-2022 (probably earlier) there’s a native support for Docker via tramp (tramp-container.el) so the above package is not required.

Reply:
> I'll do it: what value should I use?

29.0.50

Just use git blame to find where the warning is added, then browse files at that point. You will see the README containing 29.0.50.

The EmacsWiki page is erroneous.

Reply:
Okay .. I'll fix docker-tramp for you. 

It will happen tomorrow.